src/pydicom/dataset.py

```diff
@@ -2599,6 +2601,10 @@ def save_as(
             - ``Dataset.file_meta``: if the dataset is missing any required
               *File Meta Information Group* elements then they will not be
               added or written
+        exist_ok : bool, optional
```

Comment:
I also thought that `exist_ok` would be a better name, as it is already used elsewhere :)

Reply:
Funny - I had to actually go check my comment in the issue - at one point while drafting it I also had `exist_ok` as an alternate name suggestion, but I took it out.  I had a small grammatical objection because in this case it should be `exists_ok`; unlike in `makedirs` where it applies to (possibly) plural replacements, here it is always singular.  But I can live with it for the familiarity for Python users.